bookkeeper-server/src/main/java/org/apache/bookkeeper/client/DefaultBookieAddressResolver.java

```diff
@@ -59,13 +59,13 @@ public BookieSocketAddress resolve(BookieId bookieId) {
                 log.debug("Resolving dummy bookie Id {} using legacy bookie resolver", bookieId);
                 return BookieSocketAddress.resolveDummyBookieId(bookieId);
             }
-            log.info("Cannot resolve {}, bookie is unknown", bookieId, ex);
+            log.debug("Cannot resolve {}, bookie is unknown", bookieId, ex.toString());
             throw new BookieIdNotResolvedException(bookieId, ex);
         } catch (Exception ex) {
             if (ex instanceof InterruptedException) {
                 Thread.currentThread().interrupt();
             }
-            log.info("Cannot resolve {} ", bookieId, ex);
+            log.debug("Cannot resolve {} ", bookieId, ex);
```

Comment:
Same here.

Reply:
we are appending the exception to the chain of the thrown BookieIdNotResolvedException, so we are going to report this error twice (here and in the caller).

Reply:
In this case here, we would be logging messages like the one of line 47 above at debug level, but at the same time we are rethrowing, so it is possible that this log message here is unnecessary altogether.

Reply:
Quick question: since we are re-throwing, should we simply eliminate this log message altogether and let it be logged upstream?

Reply:
Right. I will remove this line at all.
Less noise is better while debugging